bin/cli.js

```diff
@@ -156,16 +157,20 @@ if (opts.slow) {
 }
 
 // load the user's tests
-var tests = []
+var tests = [],
+    extraArgs = cli.parse((fs.read(optsFile).split('\n'))).args;
```

Comment:
You're assuming the `optsFile` exists here, and that it has a test directory specified, thus breaking all the tests.